eng/scripts/Fetch-Namespaces-DotnetNupkg.ps1

```diff
@@ -5,38 +5,40 @@
   .DESCRIPTION
   The scripts is used for docs.ms to fetch namespace list for particular nupkg artifact.
 
-  .PARAMETER DotnetNupkgLocation
+  .PARAMETER DotnetNupkgLocations
   Specifies the fo nupkg location.
 
   .PARAMETER DocRepoLocation
   Specifies Location of the root of the docs.microsoft.com reference doc location. 
-
-  .PARAMETER ArtifactName
-  The artifact name. E.g. Azure.Template
 #>
 [CmdletBinding()]
 param (
-  [string] $DotnetNupkgLocation,
-  [string] $DocRepoLocation,
-  [string] $ArtifactName
+  [string] $DotnetNupkgLocations,
+  [string] $DocRepoLocation
 )
+
 Set-StrictMode -Version 3
 . (Join-Path $PSScriptRoot ../common/scripts/common.ps1)
+$ErrorActionPreference = 'Continue'
 
-Write-Host "DotnetNupkgLocation: $DotnetNupkgLocation/$ArtifactName"
-Write-Host "DocRepoLocation: $DocRepoLocation"
-
-$nupkgFile = Get-dotnet-Package-Artifacts -Location "$DotnetNupkgLocation/$ArtifactName"
-$version = $nupkgFile.Name -replace "$([Regex]::Escape($ArtifactName))\.(.*\d)\.nupkg", '$1'
-Write-Host "The full version: $version"
-$originalVersion = [AzureEngSemanticVersion]::ParseVersionString($version)
-Write-Host "The original version: $originalVersion"
-$metadataMoniker = 'latest'
-if ($originalVersion -and $originalVersion.IsPrerelease) {
-  $metadataMoniker = 'preview'
+$nupkgInfos = @($DotnetNupkgLocations | ForEach-Object { GetPackageInfoJson $_ })
+foreach($nupkgLocation in $nupkgInfos) {
+  $artifactName = Split-Path $nupkgLocation -Leaf
+  Write-Host "Nupkg Location: $nupkgLocation"
+  Write-Host "DocRepoLocation: $DocRepoLocation"
+  
+  $nupkgFile = Get-dotnet-Package-Artifacts -Location "$nupkgLocation"
+  $version = $nupkgFile.Name -replace "$artifactName.(.*\d).nupkg", '$1'
```

Comment:
You should still `[Regex]::Escape` this or any `.` could be a problem (unlikely, but possible).

Reply:
Sorry to miss this when pull the upstream